backend/venv/Lib/site-packages/click/testing.py

```
@@ -0,0 +1,565 @@
+        utils.should_strip_ansi = should_strip_ansi  # type: ignore
+        _compat.should_strip_ansi = should_strip_ansi
+
+        old_env = {}
+        try:
+            for key, value in env.items():
+                old_env[key] = os.environ.get(key)
+                if value is None:
+                    try:
+                        del os.environ[key]
+                    except Exception:
+                        pass
```

Ruff (0.12.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)